src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -1,8 +1,19 @@
 package core.basesyntax;
 
-/**
- * Feel free to remove this class and create your own.
- */
 public class HelloWorld {
```

Comment:
According to the checklist, the class implementation should not start with an empty line. Please remove the empty line at the beginning of the class definition.